src/core/SampleClip.cpp

```diff
@@ -112,6 +112,11 @@ void SampleClip::changeLength( const TimePos & _length )
 	Clip::changeLength(std::max(static_cast<int>(_length), 1));
 }
 
+void SampleClip::changeLengthToSampleLength()
+{
+	int length = static_cast<int>(m_sample.sampleSize() / Engine::framesPerTick());
```

Comment:
Do we need the static_cast if we're already assigning to int? Or is this to suppress a warning?

Reply:
You're right. We don't need it. It was "inherited" from the existing code and I moved it without putting too much thought into it. Removed with commit ef681e1b79a.